tesseract_time_parameterization/cmake/tesseract_time_parameterization-config.cmake.in

```diff
@@ -8,8 +8,10 @@ include(CMakeFindDependencyMacro)
 find_dependency(tesseract_common)
 find_dependency(tesseract_command_language)
 find_dependency(Eigen3)
-find_dependency(ruckig)
 find_dependency(console_bridge)
+if(TESSERACT_BUILD_RUCKIG)
```

Comment:
Based on how you did tesseract_motion_planners I believe this should be `if(@TESSERACT_BUILD_RUCKIG@)`

Reply:
This is most likely why it is failing to find the ruckig::ruckig target.

Reply:
I pushed a commit which fixed the build issue so the commit for `Set cmake policy....` can be removed.

Reply:
> I pushed a commit which fixed the build issue so the commit for `Set cmake policy....` can be removed.

I'm not sure that's true. Per [the cmake website](https://cmake.org/cmake/help/latest/policy/CMP0079.html) That policy is required when you call `target_link_libraries` on a target in a different directory than the directory in which the target was defined. That's what I'm doing in the sub-directories in order to keep the single target for the time parameterization library

Reply:
I seen now. Is there any reason to not have individual targets for each instead of adding them to the same one?

Reply:
Just trying to keep this change as minimally invasive as possible. It's just not working on Bionic because it's using an older version of cmake

Reply:
I would not worry about it and just create a new target for each implementation.